README.md

```diff
@@ -1,22 +1,91 @@
+# Fotowoltaika LP
+
+## PL
+
 Wtyczka pozwala na wyznaczanie miejsc lokalizacji dla farm fotowoltaicznych na gruntach w zarządzie Lasów Państowych.
 
-## Wyznaczone obszary muszą spełnić następujące warunki:
+### Wymagania:
+Warunkiem koniecznym do prawidłowego działania wtyczki jest posiadanie wersji QGIS 3.2.3 lub wyższej.
+#### Warstwy pochodne niezbędne do działania wtyczki:
+- pow_pol.shp
+- wydz_pol.shp
+- kom_lin.shp
+- oddz_pol.shp
+- ow_pkt.shp
+- nadl_pol.shp <br>
+
+Przykładowy zestaw warstw pochodnych jest rozmieszczony w rozdziale "Releases" na Github.
+
+### Instrukcja pobrania:
+1. Wtyczkę należy zainstalować w QGISie jako ZIP bądź wgrać pliki wtyczki do lokalizacji C:\Users\User\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins.
+2. Aby uruchomić wtyczkę należy kliknąć na ikonę fotowoltaiki ze słońcem.
+3. Jeżeli ikona wtyczki nie jest widoczna w panelu warstw, spróbuj zrestartować QGIS.
+4. Jeżeli wtyczka nadal nie jest widoczna  należy przejść w QGIS Desktop do Wtyczki -> Zarządzanie wtyczkami -> Zainstalowane -> Fotowoltaika LP -> Odinstalować wtyczkę, i zainstalować ponownie.<br>
+
+### Instrukcja użytkowania:
+1. W oknie wtyczki, w sekcji Wczytywanie danych, kliknij przycisk "WCZYTAJ WARSTWY POCHODNE". Otworzy się okno wyboru folderu zip z warstwami pochodnymi dla danego 
+nadleśnictwa, pobranym z Systemu Informatycznego Lasów Państwowych. 
+2. Po wybraniu folderu zip z warstwami pochodnymi do projektu zostaną załadowane warstwy z i drogami leśnymi i wydzieleniami leśnymi danego nadleśnictwa, a także mapa bazowa – Rastrowa Mapa Topograficzna Polski. 
+3. W oknie wtyczki, w sekcji Wczytywanie danych, kliknij przycisk "POBIERZ I WYŚWITL DANE BDOT10K".
+4. Do projektu zostanie załadowana warstwa linii energetycznych i dróg z BDOT10k dla powiatów, w obrębie których znajdują się wydzielenia leśne danego nadleśnictwa. Ładowanie danych może potrwać kilka minut.
+5. Po załadowaniu ww. warst można przystąpić do wykonania analizy na potrzeby farm fotowoltaicznych. Należy kliknąć przycisk "WYKONAJ ANALIZĘ NA POTRZEBY FOTOWOLTAIKI W LP". Analiza może potrwać kilka minut.
+6. Do projektu zostaną załadowane warstwy z wyznaczonymi obszarami, a także warstwy z najbliższymi liniami energetycznymi i najbliższymi drogami.
+7. Wtyczka pozwala na zapisanie warstw z wyznaczonymi obszarami, najbliższymi liniami energetycznymi i najbliższymi drogami do pliku Shapefile po kliknięciu przycisku "ZAPISZ WARSTY" i wybraniu lokalizacji zapisu w oknie dialogowym.
+8. Wtyczka umożliwia również wygenerowanie dokumentu w formacie pdf lub obrazu rastrowego po kliknięciu przycisku "GENERUJ WYDRUK" i zapisaniu go w lokalizacji wybranej w oknie dialogowym.
+9. Możliwe jest również wygenerowanie raportu (plik XLSX) z tabelarycznym wykazem wyznaczonych obszarów oraz podanymi odległościami od najbliższych dróg i linii energetycznych wraz z ich rodzajem. W tym celu należy kliknąć przycisk "GENERUJ RAPORT".
```

Comment:
To bym skumulował do jednego punktu w którym byłyby myślniki na kolejne funkcjonalności z tego działu, bo można to zrobić najpierw, a później generować raport albo na odwrót, czysta dowolność